requirements/requirements-extras-r.txt

```diff
@@ -1 +1 @@
-rpy2>=2.9.*,<3.*
\ No newline at end of file
+rpy2>=3.5.*
```

Comment:
I think it's always better to have upper bounds to dependencies versions. Assuming rpy2 follows semantic versioning, then `~=3.5` should be enough. Otherwise `~=3.5.7`, but that will require manual intervention when `3.6.0` is released if we want to use it.

Reply:
Do we have to pin to `3.5` or can we just pin to `3.*`?

Reply:
I think `3.*` is fine

Reply:
Fixed